### PR TITLE
UUID fixes

### PR DESCRIPTION
**Pull request by ZuzaOsa**

### CODE REVIEW

File: src/result.rs [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+    }
+
+    #[napi]
+    pub fn get_time_uuid(&self) -> napi::Result<TimeUuidWrapper> {
+        match self.inner.as_timeuuid() {
+            Some(r) => Ok(TimeUuidWrapper::from_cql_time_uuid(r)),
+            None => Err(Self::generic_error("time_uuid")),
+        }
+    }
+}
```
Comment (Stapox35) [inline src/result.rs:302]: Different order than in an enum.

File: lib/types/time-uuid.js [hunk 2]
```diff
@@ -378,8 +378,8 @@ function getOrGenerateRandom(id, length, callback) {
  * @private
  * @param {Date} date
  * @param {Number} ticks
- * @param {String|Buffer} nodeId
- * @param {String|Buffer} clockId
+ * @param {string|Buffer} nodeId
```
Comment (PiotrJunior) [inline lib/types/time-uuid.js:381]: if we are writing `string` in lower case then we should start writning `number` in lower case for consistency

**Review by PiotrJunior — CHANGES_REQUESTED** [review]

File: lib/types/uuid.js [hunk 3]
```diff
@@ -18,6 +18,10 @@ const uuidRegex =
  * Represents an immutable universally unique identifier (UUID). A UUID represents a 128-bit value.
  */
 class Uuid {
+    /**
+     * @type {rust.UuidWrapper}
+     * @private
+     */
```
Comment (PiotrJunior) [inline lib/types/uuid.js:24]: Add proper description for people developing this code in the future

**Review by PiotrJunior — CHANGES_REQUESTED** [review]

File: lib/types/uuid.js (hunk 3 above)
Comment (ZuzaOsa) [reply lib/types/uuid.js:24]: I do not think a comment would add a lot more decription that what can be extracted from JSDocs for this functon.

**Review by Stapox35 — APPROVED** [review]

**Pull request merged**